elastic/integrations
# [vSphere] Add vSphere package

## What does this PR do?

Adds initial vSphere package (copied from Metricbeat module).  Added syslog input for vCenter logs.

## Checklist


## Author's Checklist


## How to test this PR locally
```
cd integrations/packages/pfsense
elastic-package check && elastic-package stack down && elastic-package build && elastic-package stack up -d -v && eval "$(elastic-package stack shellinit)" && elastic-package test  -v
```

## Related issues

- Closes #642 

## Screenshots
Dashboards copied from Metricbeat module
![metricbeat_vsphere_vm_dashboard](https://user-images.githubusercontent.com/13125104/132236728-05db99eb-7c72-473f-a70f-4f834efe72a2.png)
![metricbeat_vsphere_dashboard](https://user-images.githubusercontent.com/13125104/132236731-d53d02bd-0d32-450c-8754-f8193f667366.png)
